client/batch_sql_translator.py

```diff
@@ -54,11 +55,11 @@ def start_translation(self):
         local_input_dir = self.config.input_directory
         local_output_dir = self.config.output_directory
         tmp_dir = join(dirname(self.config.input_directory), self.__TMP_DIR_NAME)
```

Comment:
is tmp_dir dead outside the conditional? Should it be pushed down?

Reply:
The tmp_dir is also used in the cleanup step. But I moved it to a member variable, hoping to reduce confusion.